README.md

```diff
@@ -0,0 +1,25 @@
+# eCommerce Bookstore
+
+## Description
+
+This is an incomplete eCommerce bookstore built with RUAH MVC PHP framework. The framework itself was developed by Curtis Parham. https://bitbucket.org/parhamcurtis/ruah-mvc-framework/src/master/ 
+
+## Prerequisites/Development Setup
+
+- To view the site on your local machine install one of the AMP solution stacks (XAMPP, LAMP, MAMP etc.) depending on your OS or install required software (Apache, MySQL/MariaDB and PHP). 
+- Create database named 'asimov' and import asimov.sql file
+
+## Configuration
+
+- In order for the contact us page to work you need to provide SMTP_USERNAME and SMTP_PASSWORD in the .env file for SMTP authentication.  
+   		
+## Todo
+
+- cart 
+- payment gateway 
+- forgotten password 
+- and more..    	
```

Comment:
"and more.." what? Give a precise list. You should update that list periodically.